data/ses/hornet.xml

```diff
@@ -286,7 +286,7 @@
           <partition_id config:type="integer">131</partition_id>
           <partition_nr config:type="integer">4</partition_nr>
           <resize config:type="boolean">false</resize>
-          <size>500M</size>
+          <size>max</size>
```

Comment:
max for boot partition?

Reply:
yes

Reply:
It will be less than 500, doesn't really matter for master

Reply:
ok then :) - I thought it's a typo

Reply:
No, I used this way with max for /boot on all nodes.
Just to be somehow consistent in all profiles \^\^